server/controllers/item-controller.js

```diff
@@ -2,240 +2,197 @@
-    // console.log(req);
-    // console.log('----------------------- createItem: body -----------------------')
-    // console.log(body);
-
-    if (!body) {
-        return res
-            .status(400)
-            .json({
-                success: false,
-                error: 'You must provide an item.',
-            });
-    }
+  const body = req.body;
+  // console.log('----------------------- createItem: req -----------------------')
+  // console.log(req);
+  // console.log('----------------------- createItem: body -----------------------')
+  // console.log(body);
+
+  if (!body) {
+    return res.status(400).json({
+      success: false,
+      error: 'You must provide an item.',
+    });
+  }
 
-    const item = new Item(body);
+  const item = new Item(body);
 
-    if (!item) {
-        console.error(`[Hack.Diversity React Template] - 400 in 'createItem': 'item' is malformed.`);
-        return res
-            .status(400)
-            .json({
-                success: false,
-                message: "'item' is malformed"
-            });
-    }
-
-    // console.log('----------------------- createItem: item -----------------------')
-    // console.log(item);
-
-    return item
-        .save()
-        .then(() => {
-            console.error(`[Hack.Diversity React Template] - 201 in 'createItem': Item created!`);
-            return res
-                .status(201)
-                .json({
-                    success: true,
-                    id: item._id,
-                    message: 'Item created!',
-                });
-        })
-        .catch(err => {
-            console.error(`[Hack.Diversity React Template] - caught error in 'createItem': ${err.errors.name}`);
-            Object.keys(err.errors).forEach(errorKey => {
-                console.error(`ERROR for: ${errorKey}`);
-                console.error(`=> ${((err.errors[errorKey] || {}).properties || {}).message}`);
-            })
-            return res
-                .status(400)
-                .json({
-                    success: false,
-                    error: err.errors,
-                    message: err.errors.name,
-                });
-        });
+  if (!item) {
+    console.error(`[Hack.Diversity React Template] - 400 in 'createItem': 'item' is malformed.`);
+    return res.status(400).json({
+      success: false,
+      message: "'item' is malformed",
+    });
+  }
+
+  // console.log('----------------------- createItem: item -----------------------')
+  // console.log(item);
+
+  return item
+    .save()
+    .then(() => {
+      console.error(`[Hack.Diversity React Template] - 201 in 'createItem': Item created!`);
+      return res.status(201).json({
+        success: true,
+        id: item._id,
+        message: 'Item created!',
+      });
+    })
+    .catch(err => {
+      console.error(
+        `[Hack.Diversity React Template] - caught error in 'createItem': ${err.errors.name}`,
+      );
```

Comment:
minor: why have this on 3 lines but not do the same for `console.error` on line 93? Whatever you decide, I would just be consistent (and to hell with the whole "under 120" characters shenanigans lol)